src/Models.jl

```diff
@@ -6,6 +6,7 @@ export Model, Template
 export fit, predict, submodels, estimate_type, output_type
 export EstimateTrait, PointEstimate, DistributionEstimate
 export OutputTrait, SingleOutput, MultiOutput
+export InputTrait, PointInput, DistributionInput
```

Comment:
I'm wondering if `InputTrait` is a confusing name since it's a different thing from the `OutputTrait` (it's more like the `EstimateTrait`)...but I can't think of a better one